### DISCUSSION TITLE
subsetArchRProject copying only Arrow files

### DISCUSSION THREAD

**Discussion by Brawni** [GreenleafLab/ArchR#571] [Feature Requests]
Hello ArchR folks!

The function subsetArchRProject copies all the folders and files from the main ArchR project folder to the new subset one. This to me seems unnecessary as you often don't need many of the files from the main project. 
Would it be possible to add an argument to the function to copy only the arrow files in the new folder? 
Also by keeping the reproduciblePeaks files from main project you would incur in this > https://github.com/GreenleafLab/ArchR/issues/407

Thanks a lot!

**Comment by jgranja24** [comment]
Yeah I will try to add something like this. Sorry for the delays!

**Comment by markphillippebworth** [comment]
So I'm subsetting an ArchRProject, and I'd like to copy everything over. But now it's only copying ArchR arrow files. 
I can recreate the project from scratch, but it'd be nice to be able to turn it off.

I'm using 1.0.2, because 1.0.1 doesn't allow for the addIterativeLSI settings that are needed for large cohorts.

**Reply by rcorces** [reply]
you mean when you run `subsetArchRProject()` it only copies the arrow files? This isnt how its supposed to work. Seems strange. It should copy all of the other files by default. The above suggestion was never implemented. `subsetArchRProject()` is just a wrapper around `saveArchRProject()`

https://github.com/GreenleafLab/ArchR/blob/968e4421ce7187a8ac7ea1cf6077412126876d5f/R/AllClasses.R#L512
